src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
 }
+
+a,
+h1,
+h2,
+h3,
+p,
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
```

Comment:
Using tag names for styling is not recommended (except `html` and `body`). It's better to use `.class`, `:pseudo-class`, `pseudo-element` and `[attribute]` for styling. Please, replace tag names with appropriate class names.